ambry-api/src/main/java/com.github.ambry/config/HelixAccountServiceConfig.java

```diff
@@ -23,6 +23,15 @@
   public static final String UPDATER_SHUT_DOWN_TIMEOUT_MS_KEY =
       HELIX_ACCOUNT_SERVICE_PREFIX + "updater.shut.down.timeout.ms";
   public static final String BACKUP_DIRECTORY_KEY = HELIX_ACCOUNT_SERVICE_PREFIX + "backup.dir";
+  public static final String INVALID_ZK_CLIENT_CONNECT_STRING = "";
+
+  /**
+   * The ZooKeeper server address. This config is required when using {@code HelixAccountService}, but not for
+   * {@code InMemoryUnknownAccountService}.
```

Comment:
i don't think the second part is required since this is `HelixAccountServiceConfig`

Reply:
removed.